tests/network-setups/indy/docker/README.md

```diff
@@ -0,0 +1,11 @@
+# Modified indy network docker image
+
```

Comment:
Provide some more context here about what we are starting. Say that this is a sample or reference IIN, or decentralized identity registry, of the form that is specified in the RFCs (link to the appropriate RFC), built on Hyperledger Indy (link to that too).